juniper_official/Solutions/RPM/rpm-probe-netconf.rule

```diff
@@ -0,0 +1,187 @@
+                        }
+                    }
+                }
+                /*
+                 * Defaults color to green.
+                 */
+                term no-packet-loss {
+                    then {
+                        status {
+                            color green;
+                            message "RTT packet Loss is not observed for $owner, $test-name";
+                        }
+                    }
+                }
+            }
+            trigger rpm-probe-round-trip-time-netconf {
+                frequency 90s;
+                /*
+                 * Sets color to red and sends out an anomaly notification when
+                 * RTT delay is more than the threshold.
+                 */
+                term is-rtt-more-than-threshold {
+                    when {
+                        greater-than "$probe-rt-delay" "$rtt-threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Average Round trip time $probe-rt-delay is more than the threshold for $owner, $test-name";
+                        }
+                    }
+                }
+                term rtt-dynamic-threshold-exceeds {
+                    when {
+                        equal-to "$probe-rt-delay-dt" 1 {
+                            time-range 3m;
+                        }
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "Average Round trip time $probe-rt-delay exceeded the dynamic threshold for $owner, $test-name";
+                        }
+                    }
+                }
+                /*
+                 * Defaults color to green.
+                 */
+                term no-anomaly {
+                    then {
+                        status {
+                            color green;
+                            message "Average Round trip time $probe-rt-delay is within the limit for $owner, $test-name";
+                        }
+                    }
+                }
+            }
+            /*
+             * Variables with default values. Default values can be changed
+             * while deploying playbook instance.
+             */
+            variable rpm-threshold-variable {
+                value 50000;
+                description "RPM max threshold variable. More than this value is marked as anomaly";
+                type int;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 3.1.0;
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products MX {
+                                releases 18.3R1 {
```

Comment:
RPM is very old feature. You can make supported Os from 15.1

Reply:
updated the supported release